### ISSUE TITLE
use table name instead of get_object_var(static)

### ISSUE THREAD

**Issue by manaphp** [klesun/deep-assoc-completion#123]
use a dummy method connect the table to class, is very convenient

**Comment by manaphp** [comment]
this feature is very cool,
 now can complete pdo query and get_object_var(static), you only need to glue them

**Comment by klesun** [comment]
For info, you can write:
```
\PDO::prepare('SELECT * FROM yourTableName')->fetch()['']
```
in a phpdoc
![image](https://user-images.githubusercontent.com/5202330/68482996-9a62ad80-0232-11ea-8098-eaa37d54f273.png)

Maybe you can achieve what you want by that?

You will need database configured.

**Comment by manaphp** [comment]
> For info, you can write:
> 
> ```
> @param $row = \PDO::prepare('SELECT * FROM yourTableName')->fetch()
> ```
> 
> in a phpdoc
> ![image](https://user-images.githubusercontent.com/5202330/68482996-9a62ad80-0232-11ea-8098-eaa37d54f273.png)
> 
> Maybe you can achieve what you want by that?
> 
> You will need database configured.

not user friendly

**Comment by manaphp** [comment]
> > For info, you can write:
> > ```
> > @param $row = \PDO::prepare('SELECT * FROM yourTableName')->fetch()
> > ```
> > 
> > 
> > in a phpdoc
> > ![image](https://user-images.githubusercontent.com/5202330/68482996-9a62ad80-0232-11ea-8098-eaa37d54f273.png)
> > Maybe you can achieve what you want by that?
> > You will need database configured.
> 
> not user friendly
good things need simple!!

**Comment by manaphp** [comment]
connects the table to class!!

**Comment by manaphp** [comment]
can we use php to generate the table name dynamicly?

**Comment by klesun** [comment]
Do you want something like this?
```
@param $row = Magic::dbRow(static::$tableName)
```

**Comment by manaphp** [comment]
> Do you want something like this?
> 
> ```
> @param $row = Magic::dbRow(static::$tableName)
> ```

yes

**Comment by klesun** [comment]
ok, why not

**Comment by manaphp** [comment]
> > Do you want something like this?
> > ```
> > @param $row = Magic::dbRow(static::$tableName)
> > ```
> 
> yes


> ok, why not

we can use magic:: namespace to do simple somthing

**Comment by klesun** [comment]
Released in [2019.11.11.001](https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/update/72078)
![image](https://user-images.githubusercontent.com/5202330/68554578-bd0ee500-0431-11ea-8061-1fcb6e59c5a3.png)

Please, let me know if something does not work as expected.
